### PR TITLE
Compute the Gauss Digitization of a simple closed polyline (no hole or self-intersection)

### PR DESCRIPTION
**Pull request by ngophuc**
*Thanks a lot for contributing to DGtalTools, before submitting your PR, please fill up the description and make sure that all checkboxes are checked.*

# PR Description

*your description here*

# Checklist

- [ ] Doxygen documentation of the code completed (classes, methods, types, members...).
- [ ] Main tool doxygen documentation (following existing documentation of [DGtalTools documentation](http://dgtal.org/doc/tools/nightly/).
- [ ] Check if it follows the tools structure described in [CONTRIBUTING.md](https://github.com/DGtal-team/DGtalTools/blob/master/CONTRIBUTING.md)
- [ ] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtalTools/blob/master/ChangeLog.md) added.
- [ ] Update the readme with potentially a screenshot of the tools if it applies. 
- [ ] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).

### CODE REVIEW

**Comment by dcoeurjo** [comment]
This feature should probably be useful in the DGtal lib, rather than in a tool, what do you think?

**Comment by ngophuc** [comment]
> This feature should probably be useful in the DGtal lib, rather than in a tool, what do you think?

It could be :) I saw the curve digitizer in DGtal-Tools that why I consider this as a tool. 
I will have a look to see how to make it for DGtal lib then.
Thanks for the proposition.

**Comment by kerautret** [comment]
and you can the two tool and in DGTal ! ;)

**Comment by ngophuc** [comment]
> and you can the two tool and in DGTal ! ;)

I would not say no 🙏

File: generators/2dSimplePolygonDigitizer.cpp [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+    // parse command line using CLI ----------------------------------------------
+    CLI::App app;
+    std::string inputFileName;
+    std::string outputFileName="result.pgm";
+    
+    app.description("compute the set of integer points inside the input polyline.\n Basic example:\n \t 2dSimplePolygonDigitizer  -i  inputPolyline.sdp -o contourDisplay.pgm");
+    app.add_option("-i,--input,1", inputFileName, "Input filename (freeman chain of sdp)." )
+    ->required()
+    ->check(CLI::ExistingFile);
+    app.add_option("-o,--output", outputFileName, "Output filename", true);
```
Comment (kerautret) [inline generators/2dSimplePolygonDigitizer.cpp:116]:
```suggestion
    app.add_option("-o,--output,2", outputFileName, "Output filename", true);
```

**Review by kerautret — APPROVED** [review]
Thanks for the tools, we could update it later if you integrated some feature in DGtal itself as discussed with David.

**Comment by kerautret** [comment]
You just forget to update the changeling @ngophuc

**Pull request merged**